aptos-move/e2e-move-tests/src/tests/mint_nft.rs

```diff
@@ -204,3 +204,47 @@ fn sample_mint_nft_signature() {
         mint_proof_signature2
     );
 }
+
+/// run `cargo test sample_tutorial_signature -- --nocapture`
+/// to sample a valid signature for calling `resource_account_address::minting::mint_nft()`
+/// in `aptos-move/move-exampels/mint_nft`
+#[test]
+fn sample_tutorial_signature() {
+    let mut h = MoveHarness::new();
+
+    // supply the resource_address
+    let resource_address = h.new_account_at(AccountAddress::from_hex_literal("0xcafe").unwrap());
```

Comment:
should this be the admin_account?

Reply:
it should be the resource account in our current design. we meant for the resource account to control the token minting and the admin account to update configs (expiration time, minting_enabled flag, etc)

Reply:
yeah I see why this is confusing... the resource account is used for minting the tokens, so it'd be part of the token data id since it's the creator. but the signature will be validated against the admin's public key.